### PR TITLE
Cluster auth fix

### PR DESCRIPTION
**Pull request by clintz**
### Description of the change

Added user/pass to remote_servers/default/shard/replica in values.yaml.

### Benefits

This allows for cluster authentication using the configured admin user/pass.

### Possible drawbacks

N/A

### Applicable issues

https://github.com/bitnami/charts/issues/13541

### Additional information

N/A

### Checklist
- [X ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Thanks for creating this PR. Please, note it is needed to sign the commits. Your signature certifies that you wrote the patch or otherwise have the right to contribute the material. You can find more info about this topic, in [the “Sign your work” section from the contributing guidelines](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work).

File: bitnami/clickhouse/values.yaml [hunk 1]
```diff
@@ -260,6 +260,8 @@ defaultConfigurationOverrides: |
             <replica>
                 <host>{{ printf "%s-shard%d-%d.%s.%s.svc.%s" (include "common.names.fullname" $ ) $shard $i (include "clickhouse.headlessServiceName" $) (include "common.names.namespace" $) $.Values.clusterDomain }}</host>
                 <port>{{ $.Values.service.ports.tcp }}</port>
+                <user>{{ $.Values.auth.username }}</user>
+                <password>{{ $.Values.auth.password }}</password>
```
Comment (javsalgar) [inline bitnami/clickhouse/values.yaml:264]: Could you please make it dependant of an env var? Otherwise the password will be plain

**Review by javsalgar — CHANGES_REQUESTED** [review]

File: bitnami/clickhouse/values.yaml (hunk 1 above)
Comment (clintz) [reply bitnami/clickhouse/values.yaml:264]: Yes. I will create a new PR.

**Comment by clintz** [comment]
Closing and creating a separate PR that address the plain text comment.

**Pull request closed without merging**